pkg/audit/manager.go

```diff
@@ -248,6 +251,16 @@ func (am *Manager) auditResources(ctx context.Context) ([]*constraintTypes.Resul
 			}
 
 			for _, obj := range objList.Items {
+				foundExcludedNamespace := false
+				for _, ns := range am.configMatchSet.ExcludedNamespaces[match.Audit] {
```

Comment:
We probably want to build a map[] for this once per run, so that the speed of each check is O(1).

Also, I'm guessing we need thread safety here, to handle the case where the config thread is changing the set at the same time.

Reply:
thanks for the review! updated to `map[Operation]map[string]bool`